addons/stock/tests/test_packing.py

```diff
@@ -1802,15 +1780,15 @@ def test_conditional_package_propagation(self):
                 'location_dest_id': self.customer_location.id,
             })]
         })
-        full_delivery.action_confirm()
-        full_delivery.action_assign()
-        self.assertEqual(full_delivery.move_line_ids.package_id, package, "The package should be used as source.")
-        self.assertEqual(full_delivery.move_line_ids.result_package_id, package, "If all the products in a package are to be moved, we must move the entire package.")
-        full_delivery.action_cancel()  # Cancel delivery to unreserve the package/quantity.
+        full_transfer.action_confirm()
+        full_transfer.action_assign()
+        self.assertEqual(full_transfer.move_line_ids.package_id, package, "The package should be used as source.")
+        self.assertEqual(full_transfer.move_line_ids.result_package_id, package, "If all the products in a package are to be moved, we must move the entire package.")
+        full_transfer.action_cancel()  # Cancel transfer to unreserve the package/quantity.
 
-        # Create 2 delivery picking : 10 & 20 of product each.
-        partial_deliveries = self.env['stock.picking'].create([{
-            'picking_type_id': self.warehouse.out_type_id.id,
+        # Create 2 internal picking : 10 & 20 of product each.
+        partial_transfer = self.env['stock.picking'].create([{
```

Comment:
```suggestion
        partial_transfers = self.env['stock.picking'].create([{
```
Nitpick, since there are two pickings :smile:

Reply:
Good point, thanks! Change applied :+1: